networking/ovn_kubernetes_network_provider/logging-network-policy.adoc

```diff
@@ -1,23 +1,24 @@
 :_content-type: ASSEMBLY
 [id="logging-network-policy"]
-= Logging network policy events
+= Logging for egress firewall and network policy rules
 include::_attributes/common-attributes.adoc[]
 :context: logging-network-policy
 
 toc::[]
 
-As a cluster administrator, you can configure network policy audit logging for your cluster and enable logging for one or more namespaces.
+As a cluster administrator, you can configure  audit logging for your cluster and enable logging for one or more namespaces. {product-title} produces audit logs for both egress firewalls and network policies.
 
 [NOTE]
 ====
-Audit logging of network policies is available for only the xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes cluster network provider].
+Audit logging is available for only the xref:../../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes cluster network provider].
 ====
 
 include::modules/nw-networkpolicy-audit-concept.adoc[leveloffset=+1]
 
-== Network policy audit configuration
+[id="network-policy-audit-configuration-{context}"]
+== Audit configuration
 
-The configuration for audit logging is specified as part of the OVN-Kubernetes cluster network provider configuration. The following YAML illustrates default values for network policy audit logging feature.
+The configuration for audit logging is specified as part of the OVN-Kubernetes cluster network provider configuration. The following YAML illustrates default values for the audit logging feature.
```

Comment:
s/the audit logging feature/audit logging

I think this is the only place where it is called _the audit logging feature_.